src/io/hdf/hdf_stl.h

```diff
@@ -144,6 +144,82 @@ struct h5data_proxy<std::string>
   }
 };
 
+// Specialization for vector of strings
+template<>
+struct h5data_proxy<std::vector<std::string>>
+{
+  typedef std::vector<std::string> ArrayType_t;
+  ArrayType_t& ref;
+
+  h5data_proxy<ArrayType_t>(ArrayType_t& a) : ref(a) {}
+
+  inline bool write(hid_t grp, const std::string& aname, hid_t xfer_plist = H5P_DEFAULT)
```

Comment:
To doxygen it is like this method has no documentation, intentional?

Reply:
I would think documentation for this method would go with the main definition of the class in hdf_dataproxy.h, not here in a specialization.

Reply:
Sure that should have general documentation. I take this to mean you only want the documentation below to appear in the implementation source.

Reply:
Yes, the documentation below relates to the implementation, not how it gets used or anything useful to a user of the class.